src/blocks/remote-data-no-result/render.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types = 1);
+
+use RemoteDataBlocks\Editor\DataBinding\BlockBindings;
+// Global variables provided by WordPress for block rendering:
+// $attributes (array): The block attributes.
+// $content (string): The block default content.
+// $block (WP_Block): The block instance.
+
+$should_render_empty_result = BlockBindings::should_render_empty_result( $block );
+
+// Skip the rendering if the block's results are not empty.
+if ( ! $should_render_empty_result ) {
+	return;
```

Comment:
This is a kind of esoteric use of `return` outside a function, but has the intended effect. I might amend to `return null;` since IIRC it is coerced to null anyway. 

https://www.php.net/manual/en/function.return.php

```suggestion
	return null;
```